src/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
`PascalCase` is reserved for React components, you're importing a JSON file so this should be `camelCase` as it's a normal javascript variable